vaadin-dashboard-flow-parent/vaadin-dashboard-flow-integration-tests/src/test/java/com/vaadin/flow/component/dashboard/tests/DashboardIT.java

```diff
@@ -16,4 +25,43 @@
  */
 @TestPath("vaadin-dashboard")
 public class DashboardIT extends AbstractComponentIT {
+
+    private DashboardElement dashboardElement;
+
+    @Before
+    public void init() {
+        open();
+        dashboardElement = $(DashboardElement.class).waitForFirst();
+    }
+
+    @Test
+    public void addWidgets_widgetsAreCorrectlyAdded() {
+        assertWidgetsByTitle("Widget 1", "Widget 2", "Widget 3");
+    }
+
+    @Test
+    public void addWidgetsAtIndex1_widgetIsAddedIntoTheCorrectPlace() {
+        $("button").id("add-widget-at-index-1").click();
```

Comment:
There's a `clickElementWithJs` utility you can use instead

Reply:
Done.

Reply:
I meant this shorthand `clickElementWithJs("add-widget-at-index-1");`

Reply:
Done.